corpus/templates/pages/sig.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block title %}
+{{sig.name}}
+{% endblock %}
+
+{% block content %}
+<section
+    class="hero min-h-[50vh] min-w-screen bg-gradient-to-br from-sky-50 to-blue-300 dark:from-blue-950 dark:to-base-100">
+    <div id="particles-js" class="h-[50vh] w-full"></div>
+    <div style="gap: 20px; justify-content: space-between; margin: auto;">
+    {% for society in societies_linked_to_sig %}
+        <a class="my-5" href="{{society.url}}">
+            {% if society.dark_image %}
+                <img src="{% static society.image %}" style="display: none; padding: 10px; height: 250px; width: auto;" alt="{{ society.name }} Logo" class="light-logo h-14 lg:h-24 inline" >
+                <img src="{% static society.dark_image %}" style="display: none; padding: 10px; height: 250px; width: auto;" alt="{{ society.name }} Logo" class="dark-logo h-14 lg:h-24 inline">
+            {% else %}
+                <img src="{% static society.image %}" style="padding: 10px; height:250px; width: auto;" alt="{{ society.name }} Logo" class="light-logo h-14 lg:h-24 inline" >
```

Comment:
```suggestion
                <img src="{% static society.image %}" style="padding: 10px; height:250px; width: auto;" alt="{{ society.name }} Logo" class="h-14 lg:h-24 inline" >
```

Keeping the `light-logo` class makes the image disappear when switching to dark mode.

Reply:
oh yes, did it